File: programmers/두_개_뽑아서_더하기/두_개_뽑아서_더하기.java
import java.util.*;

class Solution {
    public int[] solution(int[] numbers) {
        Set<Integer> stock = new HashSet<>();
        
        for(int i = 0 ; i < numbers.length - 1 ; i++){
            for(int j = i+1 ; j < numbers.length ; j++) {
                stock.add(numbers[i] + numbers[j]);
            }
        }
        
        return stock.stream().mapToInt(Integer::intValue).sorted().toArray();
    }
}

File: programmers/두_개_뽑아서_더하기/두_개_뽑아서_더하기.js
function solution(numbers) {
  const stock = new Set();
  for(let i = 0 ; i < numbers.length ; i += 1) {
      for(let j = i+1 ; j < numbers.length ; j += 1){
          stock.add(numbers[i]+numbers[j]);
      }
  }
  
  const answer = [...stock];
  answer.sort((a, b) => a-b);
  return answer;
}
